server/sonar-server/src/main/java/org/sonar/server/component/ws/ProjectsWs.java

```diff
@@ -25,6 +25,8 @@
 import org.sonar.api.server.ws.WebService;
 
 public class ProjectsWs implements WebService {
+  public static final String ENDPOINT = "api/projects";
```

Comment:
I think it's usefull to expose the WS endpoint as a general practice. I'm not sure if I want to update all my tests if I change the endpoint string. But I guess I could add a test which test the api endpoint directly